hawkular-accounts-feature-pack-resources/src/main/resources/subsystem-templates/hawkular-accounts-ejb3.xml

```diff
@@ -0,0 +1,91 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!--
+
+    Copyright 2015 Red Hat, Inc. and/or its affiliates
+    and other contributors as indicated by the @author tags.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<config default-supplement="default">
+  <extension-module>org.jboss.as.ejb3</extension-module>
+  <subsystem xmlns="urn:jboss:domain:ejb3:4.0">
+    <session-bean>
+           <?STATEFUL-BEAN?>
+      <stateless>
+        <bean-instance-pool-ref pool-name="slsb-strict-max-pool" />
+      </stateless>
+      <singleton default-access-timeout="5000" />
+    </session-bean>
+       <?MDB?>
+    <pools>
+      <bean-instance-pools>
+        <!-- Automatically configure pools. Alternatively, max-pool-size can be set to a specific value -->
+        <strict-max-pool name="slsb-strict-max-pool" derive-size="from-worker-pools" instance-acquisition-timeout="5" instance-acquisition-timeout-unit="MINUTES" />
+        <strict-max-pool name="mdb-strict-max-pool" derive-size="from-cpu-count" instance-acquisition-timeout="5" instance-acquisition-timeout-unit="MINUTES" />
+      </bean-instance-pools>
+    </pools>
+    <caches>
+      <cache name="simple" />
+      <cache name="distributable" aliases="passivating clustered" passivation-store-ref="infinispan" />
+    </caches>
+    <passivation-stores>
+      <passivation-store name="infinispan" cache-container="ejb" max-size="10000" />
+    </passivation-stores>
+    <async thread-pool-name="default" />
+    <timer-service thread-pool-name="default" default-data-store="default-file-store">
+      <data-stores>
+        <file-data-store name="default-file-store" path="timer-service-data" relative-to="jboss.server.data.dir" />
+      </data-stores>
+    </timer-service>
+    <remote connector-ref="http-remoting-connector" thread-pool-name="default" />
+    <thread-pools>
+      <thread-pool name="default">
+        <max-threads count="10" />
+        <keepalive-time time="100" unit="milliseconds" />
+      </thread-pool>
+    </thread-pools>
+       <?IIOP?>
+    <default-security-domain value="other" />
```

Comment:
Note to self, and possibly to my future self: if we change this to Keycloak, we might be able to get rid of the jboss-ejb.xml from the other modules.